react_app/src/actions/game.actions.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import * as ActionTypes from "actions/actionTypes";
+import gql from "graphql-tag";
+
+import ApolloClient from "apollo-boost";
+
+function getGameState(response) {
+    return {type: ActionTypes.GET_GAME_STATE, response};
+}
+
+export function openChallenge(data) {
+  return { type: ActionTypes.OPEN_CHALLENGE, challengeID: data };
+}
+
+export function closeChallenge() {
+  return { type: ActionTypes.CLOSE_CHALLENGE };
+}
+
+const client = new ApolloClient({
```

Comment:
I'd declare a single client somewhere in a seperate file (`client.js`)

Reply:
Putting this in Services/Network/ctfApolloClient